R/create.boxplot.R

```diff
@@ -193,19 +194,21 @@ create.boxplot <- function(
 	trellis.object <- lattice::bwplot(
 		x = formula,
 		data,
-		panel = function(...) {
+		panel = function(subscripts, ...) {
 
 			# add stripplot in background if requested
 			if (add.stripplot) {
 
 				panel.stripplot(
 					jitter.data = TRUE,
-					factor      = jitter.factor,
-					amount      = jitter.amount,
+					factor = jitter.factor,
+					amount = jitter.amount,
 					pch	 = points.info$pch,
 					col	 = points.info$col,
+					groups = points.info$groups,
+					subscripts = if (!is.null(points.info$groups)) subscripts else NULL,
```

Comment:
will this work with the flow-control {}, so something like { subscripts; } else { NULL; }  If so, that's preferred, otherwise all good.

Reply:
> Consider adding an example of using the feature
> 
> Also, check if any of the bugs are in related functions (create.violinplot, create.stripplot)

Related functions (stripplot, violinplot, scatterplot) work when groups are passed. The boxplot is a slightly different case because it is a combination of two plots, and the groups needed to be handled differently for the stripplot.